src/coreclr/vm/excep.cpp

```diff
@@ -5171,7 +5148,7 @@ BOOL NotifyAppDomainsOfUnhandledException(
 
         // This guy will never throw, but it will need a spot to store
         // any nested exceptions it might find.
-        SentEvent = AppDomain::OnUnhandledException(&throwable, isTerminating);
+        SentEvent = AppDomain::OnUnhandledException(&throwable);
```

Comment:
This now always passes `isTerminating==true` and that goes all the way to `METHOD__UNHANDLED_EVENTARGS__CTOR`. Perhaps something for another cleanup.